### ISSUE TITLE
SB not being added to page

### ISSUE THREAD

**Issue by twifty** [ajayyy/SponsorBlock#242]
A recent change has caused SponsorBlock to not be added to the page.

I'm using the latest version of chromium on Manjaro Linux. When I'm on a channels 'Videos' tab, clicking any of the videos, the SB control will not be added to the newly opened page/tab. Refreshing that page forces the control to be added. Back on a channels 'Videos' tab, refreshing the page before clicking a video, will also force the SB control to be added.

I've disabled all other extensions and can confirm they are not causing the issue.

Is there a 'debug' option so that I may provide further details?

**Comment by ajayyy** [comment]
Please make sure you are in 1.2.4 and not 1.2.3.

**Comment by twifty** [comment]
> Please make sure you are in 1.2.4 and not 1.2.3

OK, maybe this should be another issue (where is the version?). But, latest version here!

**Comment by ajayyy** [comment]
![image](https://user-images.githubusercontent.com/12688112/72220450-bb8b9700-351e-11ea-95c3-681367399466.png)
![image](https://user-images.githubusercontent.com/12688112/72220454-c0504b00-351e-11ea-821d-1139db8dc3a5.png)

**Comment by twifty** [comment]
"Latest version", please please tell me that means "the most recent" because if not my wife has some explaining to do!

I've cloned and installed from github, the problem persists! Has there been a big change recently?

**Comment by ajayyy** [comment]
Yes there has. I cannot reproduce this with.

Did you pull it from https://github.com/ajayyy/SponsorBlock/releases?

**Comment by ajayyy** [comment]
Latest version may not mean the most recent as the chrome web store delays updates for "review" for random amounts of time.

**Comment by twifty** [comment]
https://github.com/ajayyy/SponsorBlock/commit/5348496768b8f5ffa68019eb18fdf8c697d06d9d

Latest as in latest! After a quick look through the issues I'm going to guess #219 is the problem. I'll start installing older versions tomorrow when I have the time.

**Comment by ajayyy** [comment]
The commit you linked has not been rolled out to the web store yet.

Can you check if you are getting any errors? Right click the icon, click manage extensions and enable developer mode. Then enable "Collect errors".

![image](https://user-images.githubusercontent.com/12688112/72221158-cc8bd680-3525-11ea-989d-ded33c3f3f35.png)
![image](https://user-images.githubusercontent.com/12688112/72221161-d0b7f400-3525-11ea-9a4c-7f16aba93278.png)

**Comment by twifty** [comment]
When did I say "web store"?

**Comment by ajayyy** [comment]
[The latest release](https://github.com/ajayyy/SponsorBlock/releases/tag/1.2.4) does not include this commit yet.

**Comment by twifty** [comment]
Maybe I should have said "latest commit". Never mind, when I get some free time tomorrow, I'll step through the previous "versions/tags/releases/commits" and find where the problem was introduced. From there, hopefully, a solution will be resolved.

Enabling "developer mode" and/or viewing "console" has little effect, since both require reloading the page, upon which the extension is added.

**Comment by ajayyy** [comment]
This is for debugging reasons. This will allow you to see if there are any
error d triggering the issue so that you can send them to me.

**Comment by twifty** [comment]
A quick look, and I must say the default page has a bunch of its own errors, this is the only error related to SB:
```
Error in event handler: TypeError: SB.config.sponsorTimes.get is not a function
    at videoIDChange (chrome-extension://mnjggcdmjocbbbhaepdhchncahnbgone/content.js:249:51)
    at messageListener (chrome-extension://mnjggcdmjocbbbhaepdhchncahnbgone/content.js:75:13)
```
Like I said, tomorrow I'll have a "sober" look and find exactly where the problem resides.

**Comment by OfficialNoob** [comment]
Looks like my fault :/

**Comment by ajayyy** [comment]
I thought this behavior was fixed in 1.2.4, but maybe there is still something left?

**Comment by twifty** [comment]
Even though the browser states it's using the latest version, for some reason it was actually using a cached older version (not sure which). Not sure if this was caused by the browser itself or my system. I completely removed all extensions and reset the browser. Added the 1.2.4 version from google and it works correctly.

I did however confirm the problem persists in the 1.2.3 version. Latest master, ced3100, is also working fine.
